llrt_core/src/vm.rs

```diff
@@ -184,6 +185,44 @@ impl Vm {
 
         self.run(source, strict, global).await;
     }
+
+    pub async fn run_repl(&self) {
+        self.ctx
+            .with(|ctx| -> Result<()> {
+                let mut input = String::new();
+                println!("Welcome to llrt v{}", VERSION);
+                loop {
+                    print!("> ");
+                    std::io::Write::flush(&mut std::io::stdout())?;
+                    std::io::stdin().read_line(&mut input)?;
+                    match ctx.eval::<Value, _>(input.as_bytes()) {
+                        Ok(v) => console::log_std_out(
```

Comment:
Call `pub fn format` in console instead to get the formated string back. Then do `println` or `eprintln` (for exception) with that string.